mineral-ui/mineral-ui
# Update Flow

### Description

Update flow

_Note that this branch is based off of **618-webpack**_.  That'll be resolved up prior to merge.

### Motivation and context

Closes #619

Flow changelog is here...
https://github.com/facebook/flow/blob/master/Changelog.md

### Screenshots, videos, or demo, if appropriate

https://619--mineral-ui.netlify.com/

### How to test

1. No flow errors
2. No obscure warnings when start dev server
3. Make a flow error and note that it works and that messages are improved.
4. ¡Hear the cat meow en Español!

### Types of changes

- Other (provide details below)

Maintenance

### Checklist
